tests/integration/bond_test.py

```diff
@@ -201,6 +202,12 @@ def test_add_bond_with_port_and_ipv4(eth1_up, eth2_up, setup_remove_bond99):
 
 
 @pytest.mark.tier1
+@pytest.mark.xfail(
+    is_k8s(),
+    reason=("Test not fixed for k8s yet"),
```

Comment:
Please specify the following in all reasons: `Requires adjusts for k8s. Ref: https://github.com/nmstate/nmstate/issues/1579`

Reply:
Done